feeds/feed_test.go

```diff
@@ -0,0 +1,72 @@
+package feeds
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/xeipuuv/gojsonschema"
+)
+
+const schema_path = "../package.schema.json"
+
+type ExtendPackage struct {
+	*Package
+	NonConformingField string `json:"non_conforming_field"`
+}
+
+func TestValidation(t *testing.T) {
```

Comment:
Can we break this out into two tests? One for valid and one for invalid? This makes it clearer in the test logs when one of them fails.

Reply:
Yep makes sense, done